OpenOversight/app/static/css/openoversight.css

```diff
@@ -494,3 +494,60 @@ tr:hover .row-actions {
 .search-results .list-group-item {
     border: 0;
 }
+
+
+.console .button-explanation {
+    height:35px;
+    font-size:16px;
+    position:relative;
+    width: 91.66667%;
+    float:left;
+}
+
+.docs-data .input-explanation {
+    height:35px;
+    font-size:16px;
+    position:absolute;
+    width: 300%;
+    float:left;
+    left:-300px;
```

Comment:
hey so while the site doesn't currently look _great_ on mobile, for smaller screen sizes two of these explanations underlie other buttons which we should avoid, e.g.: 

<img width="613" alt="Screen Shot 2020-06-11 at 12 31 17 AM" src="https://user-images.githubusercontent.com/7832803/84345283-fac81d80-ab7a-11ea-9baf-c21d1005996c.png">

Check out smaller screen sizes in your browser, and we might want to tweak the positioning here a bit.

Reply:
PR now modified so that it appears decently for all window sizes.